github/dependabot_secrets.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
```

Comment:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

Any time a new file is added, it is stamped with the current year it was created in.  Thanks.

Reply:
Will do!